easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.4.0-CrayIntel-2015.11.eb

```diff
@@ -0,0 +1,46 @@
+# @author: gppezzi, lucamar (CSCS)
+easyblock = "ConfigureMake"
```

Comment:
We had a working 'manual' recipe and it was way faster to translate that into a standard `ConfigureMake`, rather than tweak the existing `QuantumESPRESSO` easyblock. 

Also, the QE guys here mentioned that there were quite some changes on build procedure for this version and since the resulting easyconfig looks good we didn't see a reason to spend time investigating if the custom easyblock could do better.

Why should we use the QE easyblock if the generic `ConfigureMake` seems to do the job in this case?